package/test/editDocs.js

```diff
@@ -1,6 +1,7 @@
-import fs from "fs";
+import { readFileSync, renameSync, writeFileSync } from "fs";
```

Comment:
Why is this file tracked via git?

Reply:
Its for editing the docs? I believe you already asked this above

Reply:
> Its to change the docs by
> 
>     * Editing the header to say "Welcome to the docs" and "join the discord server for help" etc
> 
>     * Removing unnecessary types. typedoc like to do this:
> 
> 
> ```
> ascent        | { large: string, small: string }
> ascent.large  | string
> ascent.small  | string
> ```
> 
> Instead of just
> 
> ```
> ascent        | { large: string, small: string }
> ```
> 
> So thats what the second part of the code is.
> 
> I figured this would be better than having to manually update / edit the docs every time they were compiled, so the doc script in the package.json also runs this automatically after.

It's used by the docGen workflow

Reply:
Oh, I see. Are you asking why the js files are tracked by git? It was just to eliminate having to recompile them, but I'll make a script to compile them